sidebars.yaml

```diff
@@ -27,19 +27,6 @@
     - page: docs/ledger-state-fix/index.md
```

Comment:
Looks like a mistake I made, LedgerStateFix appears twice in the left navigation. i'll fix that under separate cover.